db/migrations/postgres/000005_create_channel_member_history.up.sql

```diff
@@ -0,0 +1,10 @@
+CREATE TABLE IF NOT EXISTS channelmemberhistory (
+    channelid VARCHAR(26) NOT NULL,
+    userid VARCHAR(26) NOT NULL,
+    jointime bigint NOT NULL,
+    leavetime bigint,
```

Comment:
Who's adding the leavetime? Is it Gorp when we are passing the model struct?

Reply:
Yep, I think `JoinTime` doesn't appear in the `newSqlChannelMemberHistoryStore` gorp calls because it doesn't have the `NOT NULL` constraint, so it is created by gorp simply because it exists in the struct